packages/core/src/Modules/Cashier/Components/Error/virtual.jsx

```diff
@@ -51,14 +53,12 @@ class Virtual extends React.Component {
 Virtual.propTypes = {
     has_real_account     : PropTypes.bool,
     openRealAccountSignup: PropTypes.func,
-    toggleCashierModal   : PropTypes.func,
 };
 
 export default connect(
-    ({ client, ui }) => ({
+    ({ client }) => ({
```

Comment:
i think we need `ui` here for `ui.openRealAccountSignup`

Reply:
thanks, will add it